### DISCUSSION TITLE
docs/3.x/setup/read-replicas

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#980] [General]
# docs/3.x/setup/read-replicas

A database read replica is a read-only copy of the primary database. Replica should reflect changes in the primary database as fast as possible. This feature allows Saleor to offload the primary database by moving the read-only traffic to a separate database.

https://docs.saleor.io/docs/3.x/setup/read-replicas

**Comment by peteror** [comment]
It would be useful to list which operations (queries) use replicas - I see that there is more and more of that, but I spent quite some time figuring it out on my current (older) version.
